client/src/pages/ResourcePage/ResourcePage.js

```diff
@@ -0,0 +1,301 @@
+        }
+        this.setState({
+            fields,
+            collapseAll: !collapseAll
+        });
+    }
+
+    render() {
+        const { collapseAll, search } = this.state;
+        return (
+            <div className="page-container">
+                <div className="resources page static">
+                    <div className="static-page-body">
+                        <div className="row">
+                            <div className="col">
+                                <div className="static-page-title">
+                                    <h1>Resources and Glossary</h1>
+                                </div>
+                                <div className="static-page-text">
+                                    <p>
+                                        This page provides descriptions, formats, and examples of our system's fields for monuments and memorials, to assist you in submitting changes to our database.
+                                    </p>
+                                    <div className="collapse-all-search-container">
+                                        <div className="d-flex">
+                                            <TextSearch disableAnimation
+                                                        placeholder="Search fields..."
+                                                        value={search}
+                                                        className="form-control"
+                                                        onSearchChange={search => this.setState({search})}
+                                                        onClear={() => this.setState({search: ''})}/>
+                                            <Button variant="light" className="btn-sm collapse-btn"
+                                                    onClick={() => this.toggleCollapseAllFields()}>
+                                                {collapseAll ? 'Show All' : 'Collapse All'}
+                                            </Button>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        {this.renderFields()}
+                    </div>
+                </div>
+                <Footer/>
+            </div>
+        );
+    }
+
+    renderFields() {
+        const { search } = this.state;
+
+        const fields = [];
+        for (let fieldName in this.state.fields) {
+            if (!this.state.fields.hasOwnProperty(fieldName)) continue;
+            const field = this.state.fields[fieldName];
+            field.content = this.fieldContent[fieldName];
+            field.name = fieldName;
+            field.prettyName = fieldName.replace(/([A-Z])/g, ' $1').trim();
+            field.prettyName = field.prettyName.substring(0, 1).toUpperCase() + field.prettyName.substring(1);
```

Comment:
I think there's a `capitalize` function you can use in `string-util` or something like that